src/esp/gfx/CubeMap.cpp

```diff
@@ -120,6 +158,9 @@ Mn::PixelFormat getPixelFormat(CubeMap::TextureType type) {
     case CubeMap::TextureType::ObjectId:
       return Mn::PixelFormat::R32UI;
       break;
+    case CubeMap::TextureType::VarianceShadowMap:
+      return Mn::PixelFormat::RG32F;
+      break;
     case CubeMap::TextureType::Count:
       break;
   }
```

Comment:
While you're at it, I think everything above including the attachment constants could be wrapped in a `namespace {` to avoid potential symbol clashes and help the linker a bit?